modular_skyrat/modules/assault_operatives/code/assault_operatives_outfits.dm

```diff
@@ -0,0 +1,141 @@
+
+	if(cqc)
+		var/datum/martial_art/cqc/MA = new
+		MA.teach(H)
+
+	var/obj/item/implant/weapons_auth/W = new/obj/item/implant/weapons_auth(H)
+	W.implant(H)
+
+	H.update_icons()
+
+/datum/outfit/assaultops/cqb
+	name = "Assault Operative - CQB"
+
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/c20r,\
+		/obj/item/ammo_box/magazine/smgm45=4,\
+		)
+
+	cqc = TRUE
+
+/datum/outfit/assaultops/demoman
+	name = "Assault Operative - Demolitions"
+
+	belt = /obj/item/storage/belt/grenade/full
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/gyropistol,\
+		/obj/item/ammo_box/magazine/m75=4,\
+		/obj/item/implant/explosive/macro, \
+		/obj/item/storage/box/assaultops/demoman
+		)
+	l_hand = /obj/item/mod/control/pre_equipped/research
+
+/obj/item/storage/box/assaultops/demoman
+	name = "Assault Operative - Demolitions"
+
+/obj/item/storage/box/assaultops/demoman/PopulateContents()
+	for(var/i in 1 to 4)
+		new /obj/item/grenade/syndieminibomb(src)
+		new /obj/item/grenade/c4/x4(src)
+
+/datum/outfit/assaultops/medic
+	name = "Assault Operative - Medic"
+
+
+	glasses = /obj/item/clothing/glasses/hud/health
+	belt = /obj/item/storage/belt/medical/paramedic
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/pps,\
+		/obj/item/ammo_box/magazine/pps=2,\
+		/obj/item/storage/medkit/tactical=2,\
+		/obj/item/gun/medbeam)
+
+/datum/outfit/assaultops/heavy
+	name = "Assault Operative - Heavy Gunner"
+
+
+	back = /obj/item/mod/control/pre_equipped/elite
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/mg34/mg42,\
+		/obj/item/ammo_box/magazine/mg42=1,\
+		/obj/item/grenade/syndieminibomb)
+
+
+/datum/outfit/assaultops/assault
+	name = "Assault Operative - Assault"
+
+
+	back = /obj/item/mod/control/pre_equipped/elite
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/stg,\
+		/obj/item/ammo_box/magazine/stg=4,\
+		/obj/item/grenade/syndieminibomb=2)
+
+/datum/outfit/assaultops/sniper
+	name = "Assault Operative - Sniper"
+
+
+	back = /obj/item/mod/control/pre_equipped/elite
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/sniper_rifle/modular/blackmarket,\
+		/obj/item/ammo_box/magazine/sniper_rounds=4,\
+		/obj/item/grenade/syndieminibomb=2)
+
+/datum/outfit/assaultops/tech
+	name = "Assault Operative - Tech"
+
+	suit = /obj/item/mod/control/pre_equipped/elite/flamethrower
```

Comment:
this is probably fine, as it could be the techs thing (with the flamethrower deal)